Set up support section in docs (#612)
PyPSA/PyPSA: README.md

## Contributing and Support

We strongly welcome anyone interested in contributing to this project. If you have any ideas, suggestions or encounter problems, feel invited to file issues or make pull requests on GitHub.

-   In case of code-related **questions**, please post on [stack overflow](https://stackoverflow.com/questions/tagged/pypsa).
-   For non-programming related and more general questions please refer to the [mailing list](https://groups.google.com/group/pypsa).

> -   To **discuss** with other PyPSA users, organise projects, ask questions, share news, or chat with the community
> use the [discord server](https://discord.gg/JTdvaEBb).

Reviewer: ```suggestion
-   To **discuss** with other PyPSA users, organise projects, share news, and get in touch with the community you can use the [discord server](https://discord.gg/JTdvaEBb).
```
